payments/vipps/after-payment.md

```diff
@@ -737,92 +718,14 @@ Content-Type: application/json
-| reversal.transaction | object | The object representation of the generic [`transaction`][technical-reference-transaction]. |
-
-### Callback
-
-When a change or update from the back-end system are made on a payment or
-transaction, Swedbank Pay will perform a callback to inform the
-payee (merchant) about this update.
-Callback functionality is explaned in more detail
-[here][technical-reference-callback].
-
-```mermaid
-sequenceDiagram
-    activate Merchant
-    activate PAYEX
-    PAYEX->Merchant: POST <callbackUrl>
-    note left of Merchant: Callback by PayEx
-    Merchant-->PAYEX: HTTP response
-    deactivate PAYEX
-    deactivate Merchant
-
-    activate Merchant
-    Merchant->PAYEX: GET <Vipps payment>
-    note left of Merchant: First API request
-    activate PAYEX
-    PAYEX-->Merchant: payment resource
-    deactivate PAYEX
-    deactivate Merchant
-```
-
-### Problem messages
-
-When performing unsuccessful operations, the eCommerce API will respond with a
-problem message.
-We generally use the problem message type and status code to identify the
-nature of the problem.
-The problem name and description will often help narrow down the specifics
-of the problem.
-
-### Error types from Vipps (Init-call)
-
-All Vipps error types will have the following URI in front of type:
-`https://api.payex.com/psp/errordetail/vipps/<errorType>`
-
-{:.table .table-striped}
-| Type        | Status |            |
-| :---------- | :----- | :--------- |
-| VIPPS_ERROR | 403    | All errors |
-
-#### Error types from Vipps (Callback)
-
-All Vipps error types will have the following URI in front of type:
-`https://api.payex.com/psp/errordetail/vipps/<errorType>`
-
-{:.table .table-striped}
-| Type | Status |
-| VIPPS_DECLINED | 400 | Any status that is not YES
-
-#### Error types from Acquirer
+| Property              | Type     | Description                                                                                |
+| :-------------------- | :------- | :----------------------------------------------------------------------------------------- |
+| `payment`             | `string` | The relative URI of the payment this capture transaction belongs to.                       |
+| `reversal`            | `object` | The reversal object.                                                                       |
+| └➔&nbsp;`id`          | `string` | The relative URI of the created capture transaction.                                       |
+| └➔&nbsp;`transaction` | `object` | The object representation of the generic [`transaction`][technical-reference-transaction]. |
 
-All Vipps error types will have the following URI in front of type:
- `https://api.payex.com/psp/errordetail/vipps/<errorType>`
-
-{:.table .table-striped}
-| Type                          | Status |
-| :---------------------------- | :----- |
-| *CARD_BLACKLISTED*            | 400    |
```

Comment:
These errors aren't valid?

Reply:
I don't understand this comment?
Are you refering to the deleted section that was also in other-features.md?

Reply:
Or should problem messages be described both places?

Reply:
Ah ok, I see. No, problems belong in "Other Features" indeed.